src/components/form/fields/passwordField.jsx

```diff
@@ -162,6 +160,11 @@ function listEnabledRules(i18n, passwordPolicy) {
     }, {});
 }
 
+function getPasswordStrength(blacklist, fieldValue) {
+    const sanitized = `${fieldValue || ""}`.toLowerCase().trim();
+    return zxcvbn(sanitized, blacklist).score;
```

Comment:
I'm curious, what/who told you to use the `user_inputs` argument of `zxcvbn`? So hard to find a documentation about it!

Reply:
Thanks to @thomasreach5 🙏